### PR TITLE
Feat/realms playlists

### PR DESCRIPTION
**Pull request by micksabox**

### CODE REVIEW

**Comment by micksabox** [comment]
ready for review

File: apps/atlas/src/hooks/settling/useRealmsPlaylist.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    NOT: [
+      {
+        lastVaultTime: { equals: null },
+      },
+    ],
+  }),
+};
+
+export const resetPlaylistState = () => {
+  storage(realmPlaylistCursorKey, 0).remove();
```
Comment (tenoxious) [inline apps/atlas/src/hooks/settling/useRealmsPlaylist.ts:36]: Screen Shot 2022-08-16 at 6 06 18 PM
got this error on load

Comment (tenoxious) [reply apps/atlas/src/hooks/settling/useRealmsPlaylist.ts:36]: went away after reload. can't reproduce

**Review by tenoxious — COMMENTED** [review]
overall it looks good!

at some point we should change the favorite/cursor keys we save it localstorage to be wallet specific.

File: apps/atlas/src/hooks/settling/useRealmsPlaylist.ts [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+export const playlistQueryStrategy: Record<
+  PlaylistQueryType,
+  GetPlaylistQueryFilterFunction
+> = {
+  AllRealms: () => ({}),
+  Account: (account: string) => ({ settledOwner: { equals: account } }),
+  Ids: (ids: number[]) => ({ realmId: { in: ids } }),
+  Raidable: () => ({
+    NOT: [
```
[the remark this answers is not in the document]
Comment (micksabox) [reply apps/atlas/src/hooks/settling/useRealmsPlaylist.ts:27]: Yeah that alternative is simpler, I changed it.

**Comment by ponderingdemocritus** [comment]
closes #166

File: apps/atlas/src/hooks/settling/useRealmsPlaylist.ts (hunk 1 above)
Comment (micksabox) [reply apps/atlas/src/hooks/settling/useRealmsPlaylist.ts:36]: have not seen this before. It's clear that remove is a function that exists, maybe this was a 1-time anomaly having to do with hot module reloading or something.

**Comment by micksabox** [comment]
lgtm

**Pull request merged**